docs/src/language_servers.md

```diff
@@ -235,6 +235,23 @@ Follow installation instructions on [LSP-graphql](https://github.com/sublimelsp/
     }
     ```
 
+## Helm
+
+1. Install [helm-ls](https://github.com/mrjosh/helm-ls).
+2. (Optional & recommended) Install [yaml-language-server](https://github.com/redhat-developer/yaml-language-server).
```

Comment:
I would suggest removing step 2, and instead add a text that `helm-ls` could be used together with `LSP-yaml` (and add a link to LSP-yaml)

Reply:
Using both language servers from ST is a subpar experience because the helm LS cannot manage the YAML-specific parts on its own and the YAML LS chokes on Helm templates. 

It is a far better experience to have the helm LS also run the YAML LS in the background, which requires it to be installed and available on the PATH. There is probably a configuration option for the helm LS to specify a concrete path as well.

Reply:
I am also using the `YAML (Go)` with helm files, of course, but  they use the `.yml` extension, which is why I need to use a plugin to assign the correct syntax to template files following a specific path schema. I should probably tweak the selector.

Reply:
> Using both language servers from ST is a subpar experience because the helm LS cannot manage the YAML-specific parts on its own and the YAML LS chokes on Helm templates.
> It is a far better experience to have the helm LS also run the YAML LS in the background, which requires it to be installed and available on the PATH. There is probably a configuration option for the helm LS to specify a concrete path as well.
> I should probably also add a note that using LSP-yaml in parallel is not recommended.

That functionality isn't immediately obvious, until I have read what you wrote I had no idea. 
So yes, it would make sense to mention that in the docs.